src/main/resources/requests/AccountManagementRequests.ts

```diff
@@ -33,17 +33,17 @@ export class AccountManagementRequests {
      * @param payload The payload containing the azure accounts to request.
      * @param requester The user ID of the person requesting this.
      */
-    public async createPIAccount(payload, requester): Promise<boolean> {
+    public async createPIAccount(payload, requester): Promise<object | null> {
         try {
-            const response = await accountManagementApi.post('/account/add/pi', payload, {
+            const response = await accountManagementApi.post('/account/v2/pi', payload, {
                 headers: { 'x-issuer-id': requester },
             });
             logger.info('P&I account created');
-            return response.status === StatusCodes.CREATED;
+            return response.status === StatusCodes.CREATED ? response.data : null;
```

Comment:
Is the response body required here in the return? I had a look through and can only see it being used for a true / false check. If it's not being used, I think the existing boolean return here is clearer at showing the success / failure

Reply:
There problem with the true/false check is the backend endpoint is returning a 201 success status too if the endpoint returns ERRORED_ACCOUNTS rather than CREATED_ACCOUNTS, so there is no way to differentiate whether the account is successfully created or not